addon/mixins/run.js

```diff
@@ -208,6 +208,10 @@ export default Mixin.create({
     cancelDebounce(this._pendingDebounces, name);
   },
 
+  cancelThrottle(cancelId) {
```

Comment:
Thoughts on making this `cancelThrottle` the same as `cancelDebounce` where you can do something like `this.cancelDebounce('logMe');` ?

Reply:
So, cancelTimers uses an array, cancelDebounce uses the named object. I prefer how timers works because a) it's simpler and b) it matches the API of run.later. E.g const cancelId = Ember.run.later(fn, 100); Ember.run.cancel(cancelId)

Reply:
Gotcha thanks for the clear up!